android/engine/src/main/java/org/smartregister/fhircore/engine/task/WorkflowCarePlanGenerator.kt

```diff
@@ -141,37 +132,32 @@ constructor(
     output: CarePlan,
   ) {
     val patientId = IdType(patient.id).idPart
-    val planDefinitionId = IdType(planDefinition.id).idPart
 
     if (cqlLibraryIdList.isEmpty()) {
       loadPlanDefinitionResourcesFromDb()
     }
 
     val r4PlanDefinitionProcessor = createPlanDefinitionProcessor()
-    val carePlanProposal =
-      r4PlanDefinitionProcessor.apply(
-        IdType("PlanDefinition", planDefinitionId),
-        patientId,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        Parameters(),
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-      ) as CarePlan
+    // TODO Fix after resolving dependency issues
```

Comment:
What dependency issues are those? It seems to compile Ok if I uncomment this.

Reply:
The dependency issues are related to jacoco which we can address later on. Right now we can prioritize fixing measure evaluation and knowledge manager integration.